### PR TITLE
Add Exiffer.Rewrite.rewrite/3

### PR DESCRIPTION
**Pull request by feng19**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,27 @@
+# The directory Mix will write compiled artifacts to.
```
Comment (joeyates) [inline .gitignore]: This change seems outside the scope of the PR.

**Comment by joeyates** [comment]
Thanks for the valuable contribution!

File: lib/exiffer/rewrite.ex [hunk 2]
```diff
@@ -10,27 +10,50 @@ defmodule Exiffer.Rewrite do
...
 
+    :ok = Exiffer.Serialize.write(headers, output.io_device)
     :ok = Buffer.close(input)
     :ok = Buffer.close(output)
 
-    {:ok}
+    :ok
+  end
+
+  def set_make_and_model(source, destination, make, model) do
```
Comment (joeyates) [inline lib/exiffer/rewrite.ex:30]: It would be good to have a test for this

**Review by joeyates — CHANGES_REQUESTED** [review]

File: lib/exiffer/rewrite.ex (hunk 2 above)
Comment (feng19) [reply lib/exiffer/rewrite.ex:30]: OK, already add test for this on [here](https://github.com/joeyates/exiffer/pull/1/commits/4d67c31ec94a62e5d323fbda7883c7e636887f89)

File: .gitignore (hunk 1 above)
Comment (feng19) [reply .gitignore]: got it, check [revert .gitignore](https://github.com/joeyates/exiffer/pull/1/commits/46427f9d02da030323c7669186148f4e795edfd7)

Comment (feng19) [reply .gitignore]: Maybe create a new PR for this?

Comment (joeyates) [reply .gitignore]: The .gitignore file in this project is "opinionated". It follows the idea that an Elixir developer will have a global .gitignore with the standard exclusions, plus those which are specific to their situation (e.g. `.DS_Store` if they use macOS). This libraries only specificity is that it actually generates documentation - not all projects do, so that is explicitly handled.
